api-scripts/one-off-scripts/retrieving-datahub-datasets.py

```diff
@@ -0,0 +1,87 @@
+"""A script that retrieves and lists Public Data Hub Datasets.
+The output will contain a list of dataset titles, their creation date,
+and their placenames. This script was created to facilitate the tracking of
+dataset addition dates, as well as spatial spread of data, for Data
+Hub Analytics.
+
+To Run From data.naturalcapitalproject.stanford.edu repo:
+    python api-scripts/one-off-scripts/retrieving-datahub-datasets.py
+
+Dependencies:
+    $ mamaba install requests json
+"""
+
+import requests
+import json
+
+# Set the base URL to the Public Data Hub
+ckan_base_url = 'https://data.naturalcapitalproject.stanford.edu'
```

Comment:
This is super minor, but conventionally, module-level variables are defined in all-caps.  Kind of a nice-to-have style-wise!

```suggestion
CKAN_BASE_URL = 'https://data.naturalcapitalproject.stanford.edu'
```